file/provider/create_volume.go

```diff
@@ -183,7 +187,25 @@ func validateVolumeRequest(volumeRequest provider.Volume) (models.ResourceGroup,
 		resourceGroup.Name = volumeRequest.VPCVolume.ResourceGroup.Name
 	}
 
-	return resourceGroup, iops, nil
+	if volumeRequest.Capacity == nil {
+		return resourceGroup, iops, bandwidth, userError.GetUserError("VolumeCapacityInvalid", nil, nil)
+	}
+
+	// Minimum Capacity validation for non RFS profiles.
+	if *volumeRequest.Capacity < minSize && volumeRequest.VPCVolume.Profile.Name != vpcfile.RFSProfile {
```

Comment:
I think line number 190 if condition and this one can be combined

Reply:
In one it is empty and other it is some invalid value the error will change do we want to combine that ?